ide-support/lsp/lsp-server/src/main/java/module-info.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Helidon Language Server.
+ */
+module io.helidon.lsp.server {
+    requires java.logging;
+    requires com.fasterxml.jackson.core;
+    requires com.fasterxml.jackson.databind;
+    requires com.fasterxml.jackson.dataformat.yaml;
+    requires org.eclipse.lsp4j;
+    requires org.eclipse.lsp4j.jsonrpc;
+    requires io.helidon.build.common.maven;
+    requires io.helidon.build.common;
+    requires io.helidon.lsp.common;
+    requires io.helidon.config.metadata;
+    requires java.json;
+    requires org.apache.commons.lang3;
```

Comment:
Can we avoid using `org.apache.commons.lang3` ? A quick search in master shows that it's used as a test scoped dependency but it's not actually used anywhere.

We need to minimize the amount of added third party dependencies as the plugin (and its dependencies) is going to be distributed. Anything that is not a transitive dependency of the required framework (e.g. eclipse LSP4J) should be justified.